### PR TITLE
43 avatar developers

### PR DESCRIPTION
**Pull request by SaraBegache**
- [x] Making an avatar for developpers component

### CODE REVIEW

File: src/components/ArticleCard.jsx
Comment (ismail-benlaredj) [inline src/components/ArticleCard.jsx:1]: Great work Sara, nice usage of props,  there is some points i would like to address 

- add Link as a parent component 
- Try adjust the styles to align more closely with the design

### the design 
Screenshot 2023-10-20 at 16-21-36 Figma

### your work 

Screenshot 2023-10-20 at 16-49-12 Screenshot

File: src/components/DeveloperAvatar.jsx
Comment (ismail-benlaredj) [inline src/components/DeveloperAvatar.jsx:1]: also nice work here.. please add a description for member

File: src/components/ArticleCard.jsx
Comment (SaraBegache) [reply src/components/ArticleCard.jsx:1]: will do, thank you for the feedback!

File: src/components/DeveloperAvatar.jsx
Comment (SaraBegache) [reply src/components/DeveloperAvatar.jsx:1]: got it!

**Review by ismail-benlaredj — APPROVED** [review]

**Pull request merged**